docs/source/examples/zne_logical_rb_cirq_stim.md

```diff
@@ -0,0 +1,303 @@
+            stimcirq.cirq_circuit_to_stim_circuit(scaled_circuit)
+        )
+    scaled_stim_circuits.append(scaled_stim_circuit)
+```
+
+```{code-cell} ipython3
+mitigated_results = np.zeros((trials, len(code_distances)))
+
+for di in range(len(code_distances)):
+    for t in range(trials):
+        fac.reset()
+        for s, f in zip(scale_factors, scaled_stim_circuits[t]):
+            fac.push(
+                {"scale_factor": s},
+                stim_executor(
+                    f,
+                    p_err,
+                    p_th,
+                    code_distances[di],
+                    scale_shots(
+                        device_size, code_distances[di], base_shots, n_qubits
+                    ),
+                ),
+            )
+        mitigated_results[t, di] = fac.reduce()
+```
+
+## Plot results
+
++++
+
+Finally, plot the mean unmitigated and ZNE-mitigated expectation values, averaged over the trial circuits, at every code distance simulated. The error bars indicate the standard deviation.
+
+```{code-cell} ipython3
+:tags: [remove-output]
+
+mpl.pyplot.errorbar(
+    code_distances,
+    np.mean(mitigated_results, axis=0),
+    yerr=np.std(mitigated_results, axis=0),
+    label="Folding, m = 100",
+)
+
+mpl.pyplot.errorbar(
+    code_distances,
+    np.mean(noisy_results, axis=0),
+    yerr=np.std(noisy_results, axis=0),
+    ls="--",
+    marker="o",
+    label="Unmitigated, m = 100",
+)
+
+mpl.pyplot.plot(
+    code_distances,
+    np.ones((len(code_distances), 1)),
+    ls=":",
+    marker="*",
+    label="Ideal",
+    lw=2,
+)
+
+mpl.rc("font", family="Times New Roman")
+mpl.pyplot.title(
+    r"""Expectation value of $A=|00\rangle\langle00|$ from RB circuits acting on logical qubits""",
+    fontsize=12,
+)
+mpl.pyplot.xticks(ticks = [11, 13, 15, 17, 19, 21])
+
+mpl.pyplot.xlabel(r"Maximum code distance, $d_{i,0}$")
+mpl.pyplot.ylabel(r"Expectation value, $E$")
+
+mpl.pyplot.legend()
+mpl.pyplot.show()
+```
+
+```{figure} ../img/zne_stim.png
+---
+
+name: zne-stim-plot
+---
+Plot of the unmitigated and ZNE-mitigated expectation values obtained from executing the logical RB circuits.
+```
+
+We can see from the above plot that the ZNE-mitigated expectation values are closer to the ideal value of 1.0 at every code distance simulated. 
+The effect is more pronounced at lower code distances, which correspond to a higher logical error rate, whereas by $d = 21$ both the mitigated and unmitigated expectation values approach $1.0$. 
+
+```{note}
+Not all logical circuits can be folded, even at the circuit level.
+One alternative noise scaling method for logical qubits is scaling the code distance, which is referred to as distance-scaled ZNE or DS-ZNE {cite}`Wahl_2023_arXiv_ds_zne`.
+```
+
+## Additional information on DS-ZNE
+
+Modeling the noise as in Eq. (1), assuming the computation is operating in the fault tolerant regime, we see that logical error rate decreases as code distance increases.
```

Comment:
There is a way to get these reference to be automatically generated by using the syntax described [here](https://mystmd.org/guide/math#math-directives). Since there is only one equation here I don't think it's needed, but wanted to flag for future use.

Reply:
Suggestion applied, thanks Nate!